docs/src/man/hosting.md

```diff
@@ -642,21 +642,10 @@ https://USER_NAME.github.io/PACKAGE_NAME.jl/PackageB/dev
 https://USER_NAME.github.io/PACKAGE_NAME.jl/PackageB/stable  # Links to most recent PackageB version
 ```
 
-While they won't automatically reference one another, such referencing can be added manually (e.g. by linking to https://USER_NAME.github.io/PACKAGE_NAME.jl/PackageA/stable from the docs built for PackageB).
+While they won't automatically reference one another, such referencing can be added manually (e.g. by linking to `https://USER_NAME.github.io/PACKAGE_NAME.jl/PackageA/stable` from the docs built for PackageB).
 
 !!! warning
-    When building multiple subpackages in the same repo, unique `dirname`s must be specified in each package's `deploydocs`; otherwise, only the most recently built package for a given version over the entire monorepo will be present at https://USER_NAME.github.io/PACKAGE_NAME.jl/PackageB/vX.Y.Z, and the rest of the subpackages' documentation will be unavailable.
-
----
-
-**Final Remarks**
-
-That should be all that is needed to enable automatic documentation building. Pushing new
-commits to your `master` branch should trigger doc builds. **Note that other branches do not
-trigger these builds and neither do pull requests by potential contributors.**
```

Comment:
I think the only piece of useful information here is the

> Pushing new commits to your `master` branch should trigger doc builds. **Note that other branches do not
trigger these builds and neither do pull requests by potential contributors.**

part. Maybe we should move this nugget to the top of the page (e.g. into `Overview`)?

Reply:
We already have
> Note that the hosted documentation does not update when you make pull requests; you see
updates only when you merge to `master` or push new tags.

there. Maybe that can be

> Note that the hosted documentation does not update when you (or other contributors) make pull requests; you see
updates only when you merge to the trunk branch (typically, `master` or `main`) or push new tags.

to update it and reference other contributors